examples/validators/gatekeeper-validate/invalid-configmap/resources.yaml

```diff
@@ -32,6 +32,7 @@ kind: K8sBannedConfigMapKeysV1
 metadata:
   name: no-secrets-in-configmap
 spec:
+  enforcementAction: deny
```

Comment:
note: this is superfluous (default is "deny"), but fine.

Reply:
Removed it.